modules/ob-controller-settings.adoc

```diff
@@ -0,0 +1,54 @@
+// Ths module is included in the following assembly:
+//
+// * builds/openshift-builds/overview-openshift-builds.adoc
+
+:_content-type: REFERENCE
+[id="controller-settings_{context}"]
+= Controller settings
+
+The controller is configured on the {ocp-product-title} cluster with some default values. However, you can set or modify a few controller settings by using environment variables defined in the `controller.yaml` file.
+The following environment variables are available for use:
+
+.Controller settings
+[options="header"]
+|===
+
+| Environment variable | Description
+
+| `CTX_TIMEOUT` | Overrides the default context timeout that is used for all custom resource definition reconciliation operations.
+
+| `REMOTE_ARTIFACTS_CONTAINER_IMAGE` | Specifies the container image that is used for the `.spec.sources` remote artifacts download. By default, the environment variable uses the `busybox:latest` container image.
+
+| `GIT_CONTAINER_TEMPLATE` | Denotes the JSON representation of a container template that is used for steps that clone a Git repository. The default value is `{"image":"ghcr.io/shipwright-io/build/git:latest", "command":["/ko-app/git"], "securityContext":{"runAsUser":1000,"runAsGroup":1000}}`. The `args` and `name` properties are ignored as they are set by the controller.
+
+| `GIT_CONTAINER_IMAGE` | Denotes a custom container image for Git clone steps. If both the `GIT_CONTAINER_TEMPLATE` and `GIT_CONTAINER_IMAGE` variables specify an image, the image specified in the `GIT_CONTAINER_IMAGE` variable takes precedence.
+
+| `MUTATE_IMAGE_CONTAINER_TEMPLATE` | Denotes the JSON representation of a container template that is used for steps that mutates an image if a `Build` resource has annotations or labels defined in the output. The default value is `{"image": "ghcr.io/shipwright-io/build/mutate-image:latest", "command": ["/ko-app/mutate-image"], "env": [{"name": "HOME","value": "/tekton/home"}], "securityContext": {"runAsUser": 0, "capabilities": {"add": ["DAC_OVERRIDE"]}}}`. The `args` and `name` properties are ignored as they are set by the controller.
+
+| `MUTATE_IMAGE_CONTAINER_IMAGE` | Denotes the custom container image that is used for steps that mutates an image if a `Build` resource has annotations or labels defined in the output. If both the `MUTATE_IMAGE_CONTAINER_TEMPLATE` and `MUTATE_IMAGE_CONTAINER_IMAGE` variables specify an image, the image specified in the `MUTATE_IMAGE_CONTAINER_IMAGE` variable takes precedence.
```

Comment:
same here it is now called
IMAGE_PROCESSING_CONTAINER_IMAGE
see it here https://github.com/shipwright-io/build/blob/main/pkg/config/config.go

you need to change the references where MUTATE is invoked
MUTATE_IMAGE_CONTAINER_TEMPLATE -> IMAGE_PROCESSING_CONTAINER_TEMPLATE

MUTATE_IMAGE_CONTAINER_IMAGE -> IMAGE_PROCESSING_CONTAINER_IMAGE

Reply:
@apoorvajagtap yes please, If you can update the upstream and dowsnstream doc, would be nice